test/widgets/user_test.dart

```diff
@@ -78,5 +83,51 @@ void main() {
       check(await actualUrl(tester, avatarUrl)).isNull();
       debugNetworkImageHttpClientProvider = null;
     });
+
+    testWidgets('shows placeholder when image URL gives error', (WidgetTester tester) async {
```

Comment:
nit: omit boring type
```suggestion
    testWidgets('shows placeholder when image URL gives error', (tester) async {
```

Note that the existing tests near this one all do the same.